Web/Edubase.Services.Texuna/ChangeHistory/ChangeHistoryService.cs

```diff
@@ -28,13 +28,13 @@ public async Task<List<UserGroupModel>> GetSuggesterGroupsAsync(IPrincipal princ
         [Obsolete("Please use GetSuggesterGroupsAsync", true)]
         public async Task<List<UserGroupModel>> GetApproversGroupsAsync(IPrincipal principal)
         {
-            throw new NotImplementedException("GetApproversGroupsAsync is not implemented because it returns the exact same information as GetSuggesterGroupsAsync");
+            return await GetSuggesterGroupsAsync(principal);
```

Comment:
Calling `GetSuggesterGroupsAsync` within `GetApproversGroupsAsync` looks like a bug, but we can see from the previous implementation this is deliberate.

If we are keeping this change, perhaps we can add a comment indicating the rationale?
```csharp
            // Deliberate - suggesters returns the same as approvers (//todo: elaborate re: if this is referring to the Java API or something else)
            return await GetSuggesterGroupsAsync(principal);
```

Reply:
This should now be removed